Tests/Shared.Specs/AsyncFunctionExceptionAssertionSpecs.cs

```diff
@@ -48,6 +48,22 @@ public async Task When_async_method_throws_an_empty_AggregateException_it_should
             await act2.Should().ThrowAsync<XunitException>();
         }
 
+        [Fact]
+        public async Task When_async_method_throws_an_Exception_it_should_provide_the_message()
+        {
+            //-----------------------------------------------------------------------------------------------------------
```

Comment:
As `act` and `act2` are mutually independent, please split this into two separate tests.